concrete/src/Page/Page.php

```diff
@@ -2324,7 +2324,7 @@ protected function _duplicateAll($cParent, $cNewParent, $preserveUserID = false,
         if ($r) {
             while ($row = $r->fetchRow()) {
                 // This is a terrible hack.
-                if ($row['ptHandle'] = STACKS_PAGE_TYPE) {
```

Comment:
This makes me remember [this](https://github.com/concrete5/concrete5/pull/4444#discussion_r81389999) :wink: